keymaps/tree-view.cson

```diff
@@ -62,6 +62,7 @@
   'l': 'tree-view:expand-item'
   'left': 'tree-view:collapse-directory'
   'ctrl-[': 'tree-view:collapse-directory'
+  'ctrl-r': 'tree-view:collapse-all'
```

Comment:
This interferes with Symbols View on Windows and Linux.  I suggest not giving this a default keybinding - people can bind it themselves if they want to.